src/managing-workflow/configuring-load-balancers.md

```diff
@@ -1,15 +1,43 @@
 # Configuring Load Balancers
 
-Deis includes multiple routers scheduled to the cluster as part of the router mesh. In the event of a host failure, these routers can move hosts. Therefore, it is recommended that you configure a load balancer to operate in front of the Deis cluster to serve application traffic.
+Depending on what distribution of Kubernetes you use and where you host it, installation of Deis Workflow may automatically provision an external (to Kubernetes) load balancer or similar mechanism for directing inbound traffic from beyond the cluster to the Deis router(s).  For example, [kube-aws](https://coreos.com/kubernetes/docs/latest/kubernetes-on-aws.html) and [Google Container Engine](https://cloud.google.com/container-engine/) both do this.  On some other platforms-- Vagrant or bare metal, for instance-- this must either be accomplished manually or does not apply at all.
 
-These ports need to be open on the load balancers:
+## Idle connection timeouts
 
-* 80 (for application traffic and for API calls to the controller)
-* 443 (for secure traffic to the controller and applications, if applicable)
-* 2222 (for traffic to the builder)
+If a load balancer such as the one described above does exist (whether created automatically or manually) and if you intend on handling any long-running requests, the load balancer (or similar) may require some manual configuration to increase the idle connection timeout.  Typically, this is most applicable to AWS and Elastic Load Balancers, but may apply in other cases as well.  It does not apply to Google Container Engine, as the idle connection timeout cannot be configured there, but also works as-is.
 
-If you want to configure SSL termination on your load balancer, see [Platform SSL](platform-ssl.md).
+If, for instance, Deis Workflow were installed on kube-aws, this timeout should be increased to a recommended value of 1200 seconds.  This will ensure the load balancer does not hang up on the client during long-running operations like an application deployment.  Directions for this can be found [here](http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/config-idle-timeout.html).
```

Comment:
Responding to the grammar nit... "were" is an acceptable and typically preferred alternative to "was" in hypothetical situations (the sentence does start with, "If, for instance...")

I usually like to site that in Fiddler on the Roof, Tevye sings "If I _were_ a wealthy man..."  :smile: